lib/utils.js

```diff
@@ -227,6 +227,19 @@ var utils = module.exports = {
     }
     return false;
   },
+
+  callbackifyPromise: function(promise, callback) {
```

Comment:
Isn't the old name for this (`wrapTimeout`) better? It basically seems to take a callback and put it in a timeout block.

Reply:
hmm, it happens to use a timeout to achieve a secondary goal of letting a promise instead call a callback, right?

Reply:
Hm, agreed, but it seems like the timeout piece is kind of important here. `callbackifyPromise` reads like a promise utility function or something to me. Maybe `callbackifyPromiseWithTimeout`?

Reply:
SGTM, will do on Monday